sci-chemistry/bodr/bodr-10.ebuild

```diff
@@ -10,14 +10,11 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86 ~ppc-macos"
-IUSE=""
 
-DEPEND="dev-libs/libxslt"
-RDEPEND=""
-
-DOCS=( "THANKS" "AUTHORS" "README" "ChangeLog" )
+BDEPEND="dev-libs/libxslt"
 
 src_prepare() {
+	default
 	sed -i -e "s/COPYING//g" Makefile.* || die
 }
 
```

Comment:
We can drop the [docdir](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-128001r8) in newer EAPIs (yay!)

Reply:
Perfect :smiley: